src/__mock__/MockAccount.ts

```diff
@@ -40,5 +40,9 @@ class MockAccount extends Account {
   async createVault(params: { collateral: number; mintAmount: number; vault: Vault }): Promise<boolean> {
     return true;
   }
+
+  async dripInterest(params: { address: string; vault: Vault }): Promise<boolean> {
```

Comment:
As discussed on our call; it might be better to mock the `reach` API functions we call, instead of the SDL functions. This way we test the API code, and only mock Reach calls. This would simulate network calls, and help test the API works as expected.

@tomxbacked are you able to spend 15min today with nimi explaining this a bit more?

Reply:
:+1: